scripts/apps/users/views/edit-form.html

```diff
@@ -1,5 +1,5 @@
 <form name="userForm">
-    <div ng-if="user != null">
```

Comment:
I think we need this, otherwise if you select a user and then click on creating new one the form stays filled with selected user details.

Reply:
I added the condition only a few weeks ago so the avatar component doesn't crash when the user object is null or some properties are missing so it shouldn't be related with selected user details staying when creating a new one. At least I can't reproduce it.

![Peek 2020-03-24 12-41](https://user-images.githubusercontent.com/2076953/77421957-05211980-6dcd-11ea-8ca3-28503b527c78.gif)

Reply:
yep looks ok, I have some weird behaviour if I have developer tools opened, but when I close it it's all good

Reply:
That's interesting. For me, it works the same no matter if I have devtools opened or not. Maybe it's some extension of yours that's messing things up